code/game/verbs/ooc.dm

```diff
@@ -255,3 +255,30 @@ var/global/normal_ooc_colour = "#002eb8"
 
 		pct += delta
 		winset(src, "mainwindow.split", "splitter=[pct]")
+
+
+/client/verb/update_ping(time as num)
+	set instant = TRUE
+	set name = ".update_ping"
+	var/ping = pingfromtime(time)
+	lastping = ping
+	if(!avgping)
+		avgping = ping
+	else
+		avgping = MC_AVERAGE_SLOW(avgping, ping)
+
+
+/client/proc/pingfromtime(time)
+	return ((world.time + world.tick_lag * TICK_USAGE_REAL / 100) - time) * 100
+
+
+/client/verb/display_ping(time as num)
```

Comment:
```suggestion
/client/verb/display_ping(time)
```

Same question, Is num required here because its a verb?

Reply:
Not required, good for clarity.